dll/win32/shlwapi/assoc.c

```diff
@@ -542,12 +550,20 @@ HRESULT WINAPI AssocQueryStringByKeyW(ASSOCF cfFlags, ASSOCSTR str, HKEY hkAssoc
   hRet = AssocCreate( CLSID_QueryAssociations, &IID_IQueryAssociations, (void **)&lpAssoc );
   if (FAILED(hRet)) return hRet;
 
+#ifdef __REACTOS__
+  hRet = IQueryAssociations_Init(lpAssoc, cfFlags & SHLWAPI_DEF_ASSOCF, 0, hkAssoc, NULL);
+
+  if (SUCCEEDED(hRet))
+    hRet = IQueryAssociations_GetString(lpAssoc, cfFlags, str, pszExtra,
+                                        pszOut, pcchOut);
+#else
   cfFlags &= SHLWAPI_DEF_ASSOCF;
   hRet = IQueryAssociations_Init(lpAssoc, cfFlags, 0, hkAssoc, NULL);
 
   if (SUCCEEDED(hRet))
     hRet = IQueryAssociations_GetString(lpAssoc, cfFlags, str, pszExtra,
                                         pszOut, pcchOut);
+#endif
```

Comment:
```suggestion
```
This second IQueryAssociations_GetString was in common with your `#ifdef __REACTOS__`, so it could have been left outside.